src/chain/maker.js

```diff
@@ -29,6 +22,37 @@ export default async function createMaker(
     }
   };
 
+  if (useMcdKovanContracts) {
+    const MKR = createCurrency('MKR');
+    const IOU = createCurrency('IOU');
+    const kovanMcdAddresses = require('./addresses/kovan-mcd.json');
+    const addContracts = {};
+    for (let key in kovanMcdAddresses) {
```

Comment:
this is a really good place to use `reduce` instead of a for-loop, e.g.
```js
const addContracts = Object.keys(kovanMcdAddresses).reduce((result, key) => {
  result[key] = { address: { kovan: kovanMcdAddresses[key] } };
  return result;
}, {});

// or with lodash.reduce:
const addContracts = reduce(kovanMcdAddresses, (result, value, key) => {
  result[key] = { address: { kovan: value } };
  return result;
}, {});

// or with lodash.mapValues:
const addContracts = mapValues(kovanMcdAddresses, value => ({ address: { kovan: value } }));
```

Reply:
Nice, I think I can apply this to the config plugin as well.